### ISSUE TITLE
The utf-8-aware LANG set in image Dockerfiles is not used in Singularity containers

### ISSUE THREAD

**Issue by ylep** [brainvisa/casa-distro#5]
It seems that the value for `LANG` is not inherited correctly from the Docker image, and is overridden at some stage:
```
$ singularity inspect --environment cati_casa-test_ubuntu-16.04.simg
# Custom environment shell code should follow

    if [ -z "${LANG}" ];then export LANG=C.UTF-8;fi
    if [ -z "${PATH_INIT}" ];then export PATH=/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin;else export PATH="${PATH_INIT}";fi
    if [ -n "${PATH_PREPEND}" ];then export PATH="${PATH_PREPEND}:${PATH}";fi
    if [ -n "${PATH_APPEND}" ];then export PATH="${PATH}:${PATH_APPEND}";fi
```

Do we have a way of controlling how the environment is set during Docker->Singularity conversion?

**Comment by denisri** [comment]
Yes, it's the docker/singularity conversion code in casa-distro which generates it. Here LANG already has a value "C", so the export is not done. We did it like this because otherwise we cannot set values from the singularity commandline (env variables passed to the commandline are set first, the the runsctipt is run, so if it assigns variables unconditonally it overrides what we set on the commandline).
See in <tt>python/casa_distro/singularity.py in create_singularity_images()</tt>.
The fact is that I don't know how to set it and still allow commandline env...
